### PR TITLE
Improvement to `pspm_text`

### LINKED ISSUE #657: Extend or remove `pspm_text.m`

### PR DESCRIPTION
**Pull request by dominikbach**
Fixes #657  .

Should be reviewed AFTER #735 is merged.

Changes proposed in this pull request:
- integrates all import help texts contained in `pspm_text` into the respective `pspm_get_...` functions
- formats these help texts in the matlabbatch
- prunes unused warning texts from `pspm_text`
- renames `pspm_text` into `pspm_help_text` for future use with help items
- converts `pspm_help_text` into a function so that saving of the file `pspm_text.mat` is no longer necessary

### CODE REVIEW

File: src/pspm_init.m [hunk 1]
```diff
@@ -77,10 +78,6 @@
 % 2.3 Check PsPM version --
 pspm_vers = pspm_version('check');
 
-% 2.4 Load PsPM text --
-pspm_text([pspm_root, filesep]);
-load(fullfile(pspm_root,'pspm_text.mat'))
-
 % 2.5 Add required paths ---
```
Comment (teddychao) [inline src/pspm_init.m:81]: There is no 2.4 and two 2.7

File: src/pspm_trim.m [hunk 2]
```diff
@@ -195,17 +212,19 @@
...
             warning('ID:invalid_input', 'The value or name of the ending marker must be numeric or a string');
             return
         end
         % check if the markers are valid
-        if startmarker < 1 || g_endmarker < startmarker
+        if numel(startmarker) ~= 1 || ...
+                numel(g_endmarker) ~= 1 || ...
+                startmarker < 1 || ...
+                g_endmarker < startmarker
             warning('ID:invalid_input', 'No valid reference markers.\n'); return
```
Comment (teddychao) [inline src/pspm_trim.m:228]: Think at line 237, the `ndata` can be changed to `data_struct` if preferred...

**Review by teddychao — APPROVED** [review]
Only minor comments...

File: src/pspm_init.m (hunk 1 above)
Comment (dominikbach) [reply src/pspm_init.m:81]: fixed.

File: src/pspm_trim.m (hunk 2 above)
Comment (dominikbach) [reply src/pspm_trim.m:228]: changed

**Pull request merged**